Amazon.Ion.ObjectMapper.Test/TestObjects.cs

```diff
@@ -119,8 +119,8 @@ public static class TestObjects {
 
         public static Radio fmRadio = new Radio { Band = "FM" };
 
-        public static Teacher drKyler = new Teacher("Edward", "Kyler", "Math", DateTime.Parse("08/18/1962"));
-        public static Teacher drFord = new Teacher("Rachel", "Ford", "Chemistry", DateTime.Parse("04/29/1985"));
+        public static Teacher drKyler = new Teacher("Edward", "Kyler", "Math", DateTime.Parse("08/18/1962"), honda);
```

Comment:
I get what you're doing here. Now a Teacher has a car, but I feel like you could just create a new test object and not clutter the old ones.

Reply:
Created new test objects.